controller-server/src/main/java/com/yahoo/vespa/hosted/controller/proxy/ConfigServerRestExecutorImpl.java

```diff
@@ -78,27 +77,36 @@ public ProxyResponse handle(ProxyRequest proxyRequest) throws ProxyException {
         }
 
         ZoneId zoneId = ZoneId.from(proxyRequest.getEnvironment(), proxyRequest.getRegion());
-
-        // Make a local copy of the list as we want to manipulate it in case of ping problems.
-        List<URI> allServers = new ArrayList<>(zoneRegistry.getConfigServerUris(zoneId));
-
         StringBuilder errorBuilder = new StringBuilder();
-        if (queueFirstServerIfDown(allServers, proxyRequest)) {
-            errorBuilder.append("Change ordering due to failed ping.");
-        }
-        for (URI uri : allServers) {
+        // TODO: Use config server VIP for all zones that have one
+        if (zoneId.region().value().startsWith("aws-") || zoneId.region().value().startsWith("cd-aws-")) {
```

Comment:
This won't match "cd-us-east-1a"

Reply:
CD zones was renamed to cd-aws-us-east-1a (but that change was reverted, so you are right). I'll add it.